[KFP] Properly handle remote workflow termination status

adamdelman opened the pull request:
https://iguazio.atlassian.net/browse/ML-10283

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'mlrun:development' into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:adamdelman/mlrun into development

adamdelman committed:
[Automation] Also deploy mlrun-kfp image when deploying mlrun images

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge remote-tracking branch 'upstream/development' into development

adamdelman committed:
lint

adamdelman committed:
lint

adamdelman committed:
lint

adamdelman committed:
lint

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:adamdelman/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:adamdelman/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into development

adamdelman committed:
[KFP] Return correct status  when terminating KFP pod with `remote` engine
https://iguazio.atlassian.net/browse/ML-10283

adamdelman committed:
[KFP] Return correct status  when terminating KFP pod with `remote` engine
https://iguazio.atlassian.net/browse/ML-10283

adamdelman committed:
fix

adamdelman committed:
fix

adamdelman committed:
Merge branch 'development' of github.com:mlrun/mlrun into adamd/speed_up_docker_builds

adamdelman committed:
[Dockerfiles] Improve build caching

adamdelman committed:
fix

adamdelman committed:
[DB] Finalize postgres support (#8234)

**Refactor Database Utilities**

This PR replaces the MySQL-specific `MySQLUtil` with a unified `DBUtil`
class, supporting multiple databases (MySQL, PostgreSQL, SQLite).

**Key Changes:**
- Removed `MySQLUtil`.
- Added generic `DBUtil` and dialect-specific subclasses.
- Updated existing references and added tests.

https://iguazio.atlassian.net/browse/ML-10304

---------

Co-authored-by: Liran BG <liranbg@users.noreply.github.com>

tomerm-iguazio committed:
[RemoteModel] Add ModelProvider infrastructure + openaiprovider (#8159)

adamdelman committed:
fix

adamdelman committed:
fix

adamdelman committed:
fix

adamdelman committed:
fix

liranbg commented on dockerfiles/mlrun/Dockerfile:
```diff
@@ -81,12 +81,23 @@ RUN --mount=from=uv-image,source=/uv,target=/bin/uv \
     uv pip install --no-deps --require-hashes -r locked-requirements.txt --python-version ${MLRUN_PYTHON_VERSION}
 
-COPY . .
+COPY *requirements.txt .
+COPY dependencies.py .
+COPY pyproject.toml .
```
move dockerfile changes to a different PR

Yacouby committed:
[Images] Fix ml-base deprecation check to respect image tag (#8267)

This PR fixes an issue in the `enrich_image_url` logic where the client
version was used for the `mlrun/ml-base` deprecation check, even when
the user explicitly provided a tag in the image URL.

Jira:
https://iguazio.atlassian.net/browse/ML-10336

adamdelman committed:
fix

adamdelman committed:
Merge branch 'adamd/speed_up_docker_builds' into adamd/ML-10283

adamdelman committed:
fix

adamdelman committed:
Merge branch 'adamd/speed_up_docker_builds' into adamd/ML-10283

adamdelman committed:
fix

adamdelman committed:
fix
